code/game/objects/items/storage/book.dm

```diff
@@ -190,6 +190,20 @@ GLOBAL_LIST_INIT(bibleitemstates, list("bible", "koran", "scrapbook", "bible",
 			user.visible_message("<span class='notice'>[user] has purified the [sword]!!</span>")
 			qdel(sword)
 
+	else if(istype(A, /obj/item/soulstone) && !iscultist(user))
+		var/obj/item/soulstone/SS = A
+		to_chat(user, "<span class='notice'>You begin to exorcise [SS].</span>")
+		playsound(src,'sound/hallucinations/veryfar_noise.ogg',40,1)
+		if(do_after(user, 40, target = SS))
+			playsound(src,'sound/effects/pray_chaplain.ogg',60,1)
+			SS.usability = TRUE
+			for(var/mob/living/simple_animal/shade/EX in SS)
```

Comment:
soulstones will only ever have a shade in them outside of foolish mememins, so make it typeless.

Surprised soulstones just don't have a trapped_mob var or something

Reply:
disagree, this is fine

Reply:
reeee when would a soulstone ever have anything outside of a shade >:(